.evergreen/config.yml

```diff
@@ -2368,6 +2368,8 @@ axes:
           GCC_PATH: "/cygdrive/c/ProgramData/chocolatey/lib/mingw/tools/install/mingw64/bin"
           GO_DIST: "C:\\golang\\go1.20"
           SKIP_ECS_AUTH_TEST: true
+      # TODO(BUILD-17329): Update this to RHEL 8.7 after we add a new ECS task
```

Comment:
This will actually be Ubuntu 22.

Reply:
The ticket says "We would like to add a new task definition that targets RHEL 8." I thought the goal was to move away from Ubuntu?

Reply:
The build team pushed back because it was easier to target Ubuntu on ECS.

Reply:
We're still using Ubuntu for some things in the Python toolchain, the main point of moving to RHEL was for wide support for server versions.